src/app/frank/fd_frank_pack.c

```diff
@@ -63,6 +67,45 @@ fd_frank_pack_task( int     argc,
   ulong accum_ovrnp_cnt = 0UL;
   ulong accum_ovrnr_cnt = 0UL;
 
+  ulong bank_cnt = fd_pod_query_ulong( cfg_pod, "pack.bank-cnt", 0UL );
+  if( FD_UNLIKELY( !bank_cnt ) ) FD_LOG_ERR(( "pack.bank-cnt unset or set to zero" ));
+
+  ulong txnq_sz = fd_pod_query_ulong( cfg_pod, "pack.txnq-sz", 0UL );
+  if( FD_UNLIKELY( !txnq_sz ) ) FD_LOG_ERR(( "pack.txnq-sz unset or set to zero" ));
+
+  void * cu_est_mem = fd_wksp_pod_map( cfg_pod, "pack.cu-est-tbl" );
+  if( FD_UNLIKELY( !cu_est_mem ) ) FD_LOG_ERR(( "pack.cu-est-tbl unset" ));
+  /* We want the CU estimation table to persist beyond multiple runs of
+     frank, but it's a pain to have the init script initialize it.
+     We'll just try to join it.  If that fails, this is probably the
+     first run, so then we initialize it and re-join.  FIXME: This is a
+     gross solution.  Fix it when we determine how to train the table.
+     */
+  fd_est_tbl_t * est_tbl = fd_est_tbl_join( cu_est_mem );
```

Comment:
This is not a good idea for now for security reasons although we could relax later on. I would just remove the join code and recreate always.

Reply:
Good call. I made the change